src/waveform/renderers/waveformoverviewrenderer.cpp

```diff
@@ -326,4 +342,231 @@ void drawWaveformPartHSV(
+
+    if (mono) {
+        // mono means paint from bottom to top with l+r
+        const qreal dy = pPainter->deviceTransform().dy();
+        pPainter->resetTransform();
+        // shift y0 to bottom
+        pPainter->translate(0, 2 * dy);
+        // flip y-axis
+        pPainter->scale(1, -1);
+
+        for (int i = startVal, x = startVal / 2; i < end; i += 2, ++x) {
+            int low = pWaveform->getLow(i) + pWaveform->getLow(i + 1);
+            int mid = pWaveform->getMid(i) + pWaveform->getMid(i + 1);
+            int high = pWaveform->getHigh(i) + pWaveform->getHigh(i + 1);
+
+            // draw low band (widest)
+            if (low > 0) {
+                pPainter->setPen(lowColor);
+                pPainter->drawLine(x, 0, x, low);
+            }
+
+            // draw mid band (on top of low)
+            if (mid > 0) {
+                pPainter->setPen(midColor);
+                pPainter->drawLine(x, 0, x, mid);
+            }
+
+            // draw high band (on top of mid)
+            if (high > 0) {
+                pPainter->setPen(highColor);
+                pPainter->drawLine(x, 0, x, high);
+            }
+        }
+    } else { // stereo
+        for (int i = startVal, x = startVal / 2; i < end; i += 2, ++x) {
+            // left channel
+            int low = pWaveform->getLow(i);
+            int mid = pWaveform->getMid(i);
+            int high = pWaveform->getHigh(i);
+
+            // draw low band (widest)
+            if (low > 0) {
+                pPainter->setPen(lowColor);
+                pPainter->drawLine(x, static_cast<int>(-low), x, 0);
+            }
+
+            // draw mid band (on top of low)
+            if (mid > 0) {
+                pPainter->setPen(midColor);
+                pPainter->drawLine(x, static_cast<int>(-mid), x, 0);
+            }
+
+            // draw high band (on top of mid)
+            if (high > 0) {
+                pPainter->setPen(highColor);
+                pPainter->drawLine(x, static_cast<int>(-high), x, 0);
+            }
+
+            // right channel
+            low = pWaveform->getLow(i + 1);
+            mid = pWaveform->getMid(i + 1);
+            high = pWaveform->getHigh(i + 1);
+
+            // draw low band (widest)
+            if (low > 0) {
+                pPainter->setPen(lowColor);
+                pPainter->drawLine(x, 0, x, static_cast<int>(low));
+            }
+
+            // draw mid band (on top of low)
+            if (mid > 0) {
+                pPainter->setPen(midColor);
+                pPainter->drawLine(x, 0, x, static_cast<int>(mid));
+            }
+
+            // draw high band (on top of mid)
+            if (high > 0) {
+                pPainter->setPen(highColor);
+                pPainter->drawLine(x, 0, x, static_cast<int>(high));
+            }
+        }
+    }
+
+    if (start) {
+        *start = end;
+    }
+}
+
+void drawWaveformPartStackedLMH(
```

Comment:
a) this is not used anywhere
b) it's identical to LMH, which is already stacked?